### DISCUSSION TITLE
[docs]:  Implementation of Discussed Feature from YouTube Video in 'keploy' Repository

### DISCUSSION THREAD

**Discussion by 0hmX** [keploy/keploy#1884] [Q&A]
### What do you want to add to the docs? (please state reasons if any)

came across a YouTube video ([link](https://www.youtube.com/watch?v=23yQaY81Zho)) that mentioned the ability to generate test cases from an OpenAPI schema. However, I'm unsure how to accomplish this using the CLI. There is no information on the docs,  also the Keploy cli dose not give any information on how to do it.

### Repository

keploy

**Comment by soGit** [comment]
Now the [new video](https://youtu.be/5BDJaWMCuu0?feature=shared&t=7) even on the [main page](https://keploy.io/)

I guess that keploy does not actually provide functionality to generate tests based on an openapi spec directly.

**Comment by nehagup** [comment]
@soGit @cu8code you can generate tess using openAPI schema with Keploy Cloud. However, this feature is still in alpha and we're trying it with select users with supported use-case, hence the waitlist. 

Please signup for the waitlist if you'd want to try it. :)
